docs/swarm.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: default
+title: HiveMQ Swarm
+nav_order: 8
```

Comment:
Set this in 
- `logging.md` to 9
- 'configuration.md' to 10

so that Swarm is truly shown as 8. entry.